acceptance/bundle/resources/schemas/script

```diff
@@ -0,0 +1,22 @@
+echo "*" > .gitignore
+trace $CLI bundle deploy
+
+print_requests() {
+    jq 'select(.method != "GET" and (.path | contains("/unity")))' < out.requests.txt
+    rm out.requests.txt
+}
+
+trace print_requests
+read_state.py schemas schema1 id name catalog_name comment
```

Comment:
nit:

```suggestion
trace read_state.py schemas schema1 id name catalog_name comment
```

Same for other read state statements.

Reply:
I tried this, but it looks redundant since all the field names are also present in the output. We can add later something like "State:" prefix to output make it clear where this comes from.